learningresources/models.py

```diff
@@ -19,6 +19,18 @@
 log = logging.getLogger(__name__)
 
 
+def static_asset_basepath(asset, filename):
+    """
+    Creates folder base path for given asset.
+    """
+    return 'assets/{org}/{course_number}/{run}/{filename}'.format(
+        org=asset.course.org,
+        course_number=asset.course.course_number,
```

Comment:
And even if it did, that would still be ok actually, just creates an extra folder